apps/drec-ui/src/user/containers/modals/UserModalsCenter.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { LoginRegisterOrg } from './LoginRegisterOrg';
+import { UserRegistered } from './UserRegistered';
+
+export const UserModalsCenter: FC = () => {
+    return (
+        <>
+            <UserRegistered />
```

Comment:
Maybe I am missing something, but I was unable to find the context which is handling user related modals

Reply:
There is a reference in LoginPage.effects